src/lib/integration_test_lib/intf.ml

```diff
@@ -199,10 +197,10 @@ module Engine = struct
         -> pk:Signature_lib.Public_key.Compressed.t
         -> string list Deferred.Or_error.t
 
-      val get_peer_id :
+      val get_peer_ids :
         logger:Logger.t -> t -> (string * string list) Deferred.Or_error.t
```

Comment:
Hi. Can you tell me why this has been changed? If it's serving no functional purpose, it might be worth changing it back. I would like the PR to be as lean as possible with only changes relevant to the abstract engine. Thanks.

Reply:
This name is strictly more correct, but I'll change it back to minimize the number of changes to GCP related modules